src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const bodyTable = document.querySelector('table');
+
+const allRows = bodyTable.getElementsByTagName('tr');
+
+addRow.addEventListener('click', (e) => {
+  if (allRows.length < 10) {
+    const newRow = allRows[allRows.length - 1].cloneNode(true);
+
+    bodyTable.append(newRow);
+  }
+});
+
+removeRow.addEventListener('click', (e) => {
+  if (allRows.length > 2) {
+    allRows[allRows.length - 1].remove();
+  }
+});
+
+addColumn.addEventListener('click', (e) => {
+  if (allRows[0].children.length < 10) {
+    for (let i = 0; i < allRows.length; i++) {
+      const cell = document.createElement('td');
+
+      allRows[i].append(cell);
```

Comment:
When adding a new column, the newly created cell (`<td>`) is empty. If you intend to have default content or specific attributes, you should set them here. Otherwise, the new cells will be empty, which might not be the desired behavior.